### PR TITLE
Product carousel tests

### PR DESCRIPTION
**Pull request by r-jeng**
@afong888 
Could you help code review this? 
I added a test for ProductCarousel and added proptypes for the Price component.

### CODE REVIEW

File: tests/ProductCarousel.test.jsx [hunk 1]
```diff
@@ -97,34 +136,27 @@ jest.mock('axios');
...
+it('should load and display product styles for related product data',
+  () => axios.get('/products/48433/styles')
+    .then((relatedProductStyles) => expect(relatedProductStyles).toEqual(mockRelatedDataThird)));
 
 it('should load and display carousel module title', async () => {
   render(
     <Provider store={store}>
       <RelatedItems />
     </Provider>,
   );
```
Comment (afong888) [inline tests/ProductCarousel.test.jsx:155]: you can take out async and check for loading... when your items are loading

Comment (afong888) [reply tests/ProductCarousel.test.jsx:155]: or you can delete it as the test is not currently testing anything

**Review by afong888 — COMMENTED** [review]
Good job on tests! Left a comment on the first test

File: tests/ProductCarousel.test.jsx (hunk 1 above)
Comment (r-jeng) [reply tests/ProductCarousel.test.jsx:155]: Removed! Thanks @afong888 !

**Pull request merged**